CHANGELOG.md

```diff
@@ -0,0 +1,69 @@
+# Changelog
+
+### 0.9.1 (2015-07-20)
+
+* make default `controllerWhitelist` more specific
```

Comment:
I'm not sure if this level of detail is specific enough. If somebody reads this, do they know what has changed in v0.9.1?